### PR TITLE
Add show techsupport feature for Marvell(Innovium) platform

### PR DESCRIPTION
**Pull request by pdhruv-marvell**
Add show techsupport feature for Marvell(Innovium) platform

Signed-off-by: pdhruv-marvell pdhruvkumarr@marvell.com

#### What I did
Added support to collect Marvell (Innovium) platform logs under  "show techsupport"
#### How I did it
Enhanced generate_dump.py to collect Marvell logs when run on Marvell platforms
#### How to verify it
On SONiC device CLI:
================================
```
admin@sonic-wistron13-dut:~$ show techsupport
.
.
.
{
    cat
}; ivmcmd 'show techsupport -i /innovium/show_techsupport_infile' | dummy_cleanup_method &> '/var/dump/sonic_dump_sonic-wistron13-dut_20230428_125015/dump/show_techsupport_op_ifcs.log'"
timeout --foreground 5m bash -c "dummy_cleanup_method ()
{
    cat
}; ivmcmd 'show techsupport -i /innovium/show_techsupport_infile_iSAI' | dummy_cleanup_method &> '/var/dump/sonic_dump_sonic-wistron13-dut_20230428_125015/dump/show_techsupport_op_iSAI.log'"
timeout --foreground 5m bash -c "dummy_cleanup_method ()
{
.
.
.
```
#### Previous command output (if the output of a command-line utility has changed)
NA

#### New command output (if the output of a command-line utility has changed)
NA

### CODE REVIEW

**Comment by maipbui** [comment]
@pdhruv-marvell Please merge latest master code to trigger Semgrep.

**Comment by pdhruv-marvell** [comment]
@maipbui latest master code merged and  Semgrep check passed.

**Comment by rck-innovium** [comment]
> @maipbui latest master code merged and Semgrep check passed.

Hi @maipbui : This is a Marvell (Innovium) specific change and we have verified it on our platforms. Can you please review and merge.

File: scripts/generate_dump [hunk 1]
```diff
@@ -1412,6 +1412,20 @@ collect_cisco_8000() {
...
+# collect_innovium
+# Globals:
+#   None
+# Arguments:
+#   None
+# Retuens:
+#   None
+##############################################################################
+collect_innovium() {
+        save_cmd "ivmcmd 'show techsupport -i /innovium/show_techsupport_infile'" "show_techsupport_op_ifcs.log"
```
Comment (qiluo-msft) [inline scripts/generate_dump:1425]: >         [](http://example.com/codeflow?start=0&length=8)

To many indentation, 4-space is enough.

**Comment by pdhruv-marvell** [comment]
Hi @qiluo-msft : Review comments addressed.

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**